http/src/main/resources/reference.conf

```diff
@@ -513,7 +513,9 @@ gke {
     # Templated by firecloud-develop
     postgres.password = "replace-me"
     orchUrl = "https://firecloud-orchestration.dsde-dev.broadinstitute.org/api/"
-    drsUrl = "https://us-central1-broad-dsde-dev.cloudfunctions.net/martha_v3"
+    drsResolver = "martha"
+    marthaUrl = "https://us-central1-broad-dsde-dev.cloudfunctions.net/martha_v3"
+    drshubUrl = "https://drshub.dsde-dev.broadinstitute.org/api/v4/drs/resolve"
```

Comment:
Noting that this URL actually gets sent to Galaxy, and Galaxy makes DRS calls. I think this is the code: https://github.com/anvilproject/fs.anvilfs

Does the DRSHub API have the same shape as martha? E.g. same auth, params, etc. If so then it should be safe, but ideally we'd test with Galaxy before cutting over.

Reply:
@tlangs @andy7i can u comment on this..I thikn this is the most important thing to confirm besides making sure `fireloud-develop` is merged before this one

Reply:
based on the documentation, it looks very similar. 
https://drshub.dsde-dev.broadinstitute.org/#/drsHub/resolveDrs
https://github.com/broadinstitute/martha#martha-v3

@tlangs do you know off the top of your head whether the two APIs are functionally equivalent (for example, parameters, response, etc.)?

Reply:
They should be equivalent, as DRSHub is a direct port of Martha.

Reply:
cool..once this is deployed..can we also make sure things are working as expected in Galaxy on dev?

cc @afgane

Reply:
yeah, how would we do that? Would these be manual tests / are they written down somewhere?

Reply:
I don't actually know very well how it's being used either...was hoping @afgane could provide some more info

Reply:
ok this PR is more of a cleanup PR so I'm going to put this under [ID-347](https://broadworkbench.atlassian.net/browse/ID-347) instead. 

@afgane please let us know how to test galaxy either here, slack (#dsp-identiteam) or on https://broadworkbench.atlassian.net/browse/ID-313